client/containers/DashboardCustomScripts/DashboardCustomScripts.tsx

```diff
@@ -11,13 +12,14 @@ require('./dashboardCustomScripts.scss');
 
 type Props = {
 	customScripts: CustomScripts;
+	communityId: Community;
 };
 
 const DashboardCustomScripts = (props: Props) => {
-	const { customScripts } = props;
+	const { customScripts, communityId } = props;
```

Comment:
We have a handy `usePageContext()` hook that aggregates a lot of useful information about the page the user is currently visiting, including the Community. So you'll never have to pass that info down from the server by hand — instead you can grab the ID like this:

```ts
const { communityData: { id: communityId }} = usePageContext();
```

We should really convert `utils/hooks` to TypeScript and add a `PageContext` type to help clarify what this contains, but I encourage you to take a look around at callsites for this hook to get a sense for its various usess.